github/ci/prow-deploy/files/jobs/kubevirt/kubevirt/kubevirt-presubmits.yaml

```diff
@@ -904,6 +905,7 @@ presubmits:
       sriov-pod: "true"
     max_concurrency: 10
     name: pull-kubevirt-e2e-kind-1.19-sriov
```

Comment:
Why this is not deleted? You want to defer this to a later stage?

Reply:
I think there is no need to defer it, done